### PR TITLE
refactor: tce-broadcast config

### PR DESCRIPTION
**Pull request by Freyskeyd**
# Description

This PR aims to refactor some part of the TCE broadcast configuration.
It also remove some unused code such as `StableSample` event.

## PR Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added or updated tests that comprehensively prove my change is effective or that my feature works

### CODE REVIEW

File: crates/topos-config/src/tce/broadcast.rs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct ReliableBroadcastParams {
+    /// Echo threshold
```
Comment (dvdplm) [inline crates/topos-config/src/tce/broadcast.rs:5]: I have read the PR but it is not clear to me what these threshold actually are. Is this how many messages of a given type must be received (or sent?) to be considered sent? Or what are they?

File: crates/topos-tce-broadcast/src/event.rs [hunk 2]
```diff
@@ -1,39 +1,9 @@
...
-#[command(name = "Parameters of the reliable broadcast")]
-pub struct ReliableBroadcastParams {
-    /// Echo threshold
-    #[arg(long, env = "TCE_ECHO_THRESHOLD", default_value_t = 1)]
-    pub echo_threshold: usize,
-    /// Ready threshold
-    #[arg(long, env = "TCE_READY_THRESHOLD", default_value_t = 1)]
-    pub ready_threshold: usize,
-    /// Delivery threshold
-    #[arg(long, env = "TCE_DELIVERY_THRESHOLD", default_value_t = 1)]
```
[the remark this answers is not in the document]
Comment (Freyskeyd) [reply crates/topos-tce-broadcast/src/event.rs:21]: Those values were not parsed by the CLI parser, thus not used. The thresholds are defined by the number of validators in the genesis file now.

File: crates/topos-config/src/tce.rs [hunk 3]
```diff
@@ -83,72 +88,13 @@ pub struct TceConfig {
...
-        default_libp2p_api_addr().port()
-    )
-    .parse()
-    .expect(
-        r#"
-        Public multiaddresses generation failure.
-        This is a critical bug that need to be report on `https://github.com/topos-protocol/topos/issues`
-    "#,
-    )]
+fn default_db_path() -> PathBuf {
```
[the remark this answers is not in the document]
Comment (Freyskeyd) [reply crates/topos-config/src/tce.rs:95]: `PathBuf` needs to allocate indeed.

File: crates/topos-config/src/tce/broadcast.rs (hunk 1 above)
Comment (Freyskeyd) [reply crates/topos-config/src/tce/broadcast.rs:5]: That the thresholds of the reliable broadcast protocol.

**Pull request merged**